lang/cs/Org.Apache.REEF.Utilities/Logging/Logger.cs

```diff
@@ -107,7 +107,7 @@ public static void SetCustomLevel(Level customLevel)
             _customLevel = customLevel;
         }
 
-        public static void AddTraceListner(TraceListener listener)
```

Comment:
Public method in non-example non-test code => needs to go through "Obsolete in 0.14" path